package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "blackbaud-skyux-lib-restricted-view",
+  "version": "1.0.0",
+  "description": "Single-page-application for skyux-lib-restricted-view",
+  "main": "bundles/bundle.umd.js",
+  "module": "index.ts",
+  "engines": {
+    "node": ">=6.11.1"
+  },
+  "scripts": {
+    "skyux": "skyux",
+    "test": "skyux test && skyux e2e && skyux build && npm run build",
+    "build": "skyux build-public-library",
+    "start": "skyux serve"
+  },
+  "keywords": [],
+  "author": "Blackbaud",
+  "license": "ISC",
+  "peerDependencies": {
```

Comment:
We should be supporting SKY UX 3 with this new library.  So these dependencies will need to be adjusted to point to the required SKYUX 3 modules.   The (https://developer.blackbaud.com/skyux-migration-guide/)[migration guide] may be of some help here.